cves/kernel/CVE-2021-45095.yml

```diff
@@ -311,9 +328,9 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: true
+  note: very limited conversion on whether to classify this as a bug or vulenrability.
```

Comment:
Potentially useful to link the discussion https://lore.kernel.org/all/20211209082839.33985-1-hbh25y@gmail.com/ in this note? Not sure if this is usually what's done...

Reply:
added it to be safe